packages/build/tests/plugins/snapshots/tests.js.md

```diff
@@ -11204,28 +11194,30 @@ Generated by [AVA](https://ava.li).
       1. onPreBuild command from ./plugin                           ␊
     ────────────────────────────────────────────────────────────────␊
     ␊
-    Netlify configuration property "functions.node_bundler" value changed to 'zisi'.␊
-    Netlify configuration property "functions.external_node_modules" value changed to [ 'test' ].␊
-    Netlify configuration property "functions.ignored_node_modules" value changed to [ 'test' ].␊
-    Netlify configuration property "functions.included_files" value changed to [ 'test' ].␊
+    Netlify configuration property "functions.test.external_node_modules" value changed to [].␊
+    Netlify configuration property "functions.test.ignored_node_modules" value changed to [].␊
+    Netlify configuration property "functions.test.included_files" value changed to [].␊
     Netlify configuration property "functions.*.node_bundler" value changed to 'zisi'.␊
     Netlify configuration property "functions.*.external_node_modules" value changed to [].␊
     Netlify configuration property "functions.*.ignored_node_modules" value changed to [].␊
     Netlify configuration property "functions.*.included_files" value changed to [].␊
-    Netlify configuration property "functions.test.node_bundler" value changed to 'esbuild'.␊
-    Netlify configuration property "functions.test.external_node_modules" value changed to [].␊
-    Netlify configuration property "functions.test.ignored_node_modules" value changed to [].␊
-    Netlify configuration property "functions.test.included_files" value changed to [].␊
+    Netlify configuration property "functions.node_bundler" value changed to 'zisi'.␊
+    Netlify configuration property "functions.external_node_modules" value changed to [ 'test' ].␊
+    Netlify configuration property "functions.ignored_node_modules" value changed to [ 'test' ].␊
+    Netlify configuration property "functions.included_files" value changed to [ 'test' ].␊
     ␊
     > Updated config␊
       build:␊
         publish: packages/build/tests/plugins/fixtures/config_mutate_functions_any␊
         publishOrigin: default␊
       functions:␊
         '*':␊
-          external_node_modules: []␊
-          ignored_node_modules: []␊
-          included_files: []␊
+          external_node_modules:␊
+            - test␊
+          ignored_node_modules:␊
+            - test␊
+          included_files:␊
+            - test␊
```

Comment:
This test snapshot change is due to `functions.*` denormalization and is expected.